pkg/apis/pipeline/v1beta1/pipelinerun_validation_test.go

```diff
@@ -182,20 +292,50 @@ func TestPipelineRun_Validate(t *testing.T) {
 						}},
 						TaskSpec: &v1beta1.EmbeddedTask{TaskSpec: v1beta1.TaskSpec{
 							Params: []v1beta1.ParamSpec{{
-								Name: "task-words",
+								Name: "default-task-words",
```

Comment:
why did you rename this param?

Reply:
Same reason as above. Let me know if you need more clarification.

Reply:
the discussion above covered it, thanks